.github/workflows/error-codes-updater.yml

```
@@ -0,0 +1,63 @@
+name: Meshsync Error Codes Utility Runner
+on:
+  push:
+    branches:
+      - 'master'
+    paths:
+      - '**.go'
+
+jobs:
+  Update-error-codes:
+    name: Error codes utility
+    if: github.repository == 'meshery/meshsync'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      # token here with write access to meshsync repo
+        with:
+          token: ${{ secrets.GH_ACCESS_TOKEN }}
+          ref: 'master'
+
+      - name: Setup Go
+        uses: actions/setup-go@v1
+        with:
+          go-version: ${{ secrets.GO_VERSION }}
+
+      - name: Run utility
+        run: |
+          go run github.com/layer5io/meshkit/cmd/errorutil -d . update --skip-dirs meshery -i ./helpers -o ./helpers
```

```suggestion
          go get github.com/layer5io/meshkit/cmd/errorutil
          go run github.com/layer5io/meshkit/cmd/errorutil -d . update --skip-dirs meshery -i ./helpers -o ./helpers
```
@metonymic-smokey let's also add this here to ensure that we use a newer version of the utility on the action runner and not v2.14